### DISCUSSION TITLE
Lost functionality

### DISCUSSION THREAD

**Discussion by AnniekDev** [community/community#22229] [Other Feature Feedback, Questions, & Ideas]
<p>Hi. I made two repos so far. Both of them have the same problems. Some pictures do not show. I checked the html for the path and that seems ok. Locally it runs just fine. What can I do about this?<br>
Also the dropdown in the navbar doesn’t work anymore. How can I fix this?</p>

**Comment by AnniekDev** [comment]
<p>So, I found out the problem was the path after all. Wrote png instead of PNG. My other problem remains. There is no functionality. How can I make it work?</p>

**Comment by otter-computer** [comment]
<p>Hi <a class="mention" href="/anniekdev">@AnniekDev</a>, welcome to the GitHub Support Community! Can you post a link to your site and the repository that hosts it so we can have a look at what might be going wrong?</p>

**Comment by AnniekDev** [comment]
<p>Hi!<br>
Is this the link you need?</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/AnniekDev/climbersCampgrounds" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/73695311/181069642-511b83e9-8168-42c4-8167-f18f17f7ab94.jpeg" class="thumbnail onebox-avatar" width="400" height="400">

<h3><a href="https://github.com/AnniekDev/climbersCampgrounds" target="_blank" rel="noopener nofollow ugc">AnniekDev/climbersCampgrounds</a></h3>

<p>Contribute to AnniekDev/climbersCampgrounds development by creating an account on GitHub.</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>Thanks!</p>

**Comment by otter-computer** [comment, marked as answer]
<p>Yep, that’ll do, thanks! It looks like there’s an error in the developer console when visiting your site where the bootstrap javascript file isn’t loading. This is likely because Jekyll is still enabled for your site meaning the <code>node_modules</code> folder isn’t being included when your site is deployed.</p>
<p>To stop this you’ll need to create an empty file in the root of your repository called <code>.nojekyll</code>. This will disable Jekyll for your site and <em>should</em> mean that the <code>node_modules</code> folder is included correctly.</p>
<p>Alternatively you could include boostrap by using a hosted version from a CDN instead of a locally installed version. There’s instructions on how to do that on this page:<br>
</p><aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/2439407/181069645-c787f19a-01f8-426a-bbfd-0088b996272f.png" class="site-icon" width="32" height="32">
      <a href="https://getbootstrap.com/docs/4.5/getting-started/introduction/#css" target="_blank" rel="noopener">getbootstrap.com</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/362;"><img src="https://user-images.githubusercontent.com/2439407/181069646-8669d181-331c-4b60-a4d2-5377499a26c0.jpeg" class="thumbnail" width="690" height="362" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/5/d/5db6e51b214040a906c42f0dd103eee32e2bab63_2_690x362.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/5/d/5db6e51b214040a906c42f0dd103eee32e2bab63_2_1035x543.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/3X/5/d/5db6e51b214040a906c42f0dd103eee32e2bab63.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/5/d/5db6e51b214040a906c42f0dd103eee32e2bab63_2_10x10.png"></div>

<h3><a href="https://getbootstrap.com/docs/4.5/getting-started/introduction/#css" target="_blank" rel="noopener">Introduction</a></h3>

<p>Get started with Bootstrap, the world’s most popular framework for building responsive, mobile-first sites, with jsDelivr and a template starter page.</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by AnniekDev** [comment]
<p>Thank you so much! First option worked for me!</p>
